com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageSender.cs

```diff
@@ -1,29 +1,66 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using Unity.Profiling;
 using MLAPI.Configuration;
 using MLAPI.Internal;
 using MLAPI.Logging;
 using MLAPI.Profiling;
 using MLAPI.Security;
+using MLAPI.Serialization.Pooled;
 using BitStream = MLAPI.Serialization.BitStream;
 
 namespace MLAPI.Messaging
 {
-    internal static class InternalMessageSender
+    public static class InternalMessageSender
```

Comment:
I think this needs to be internal still :)

Reply:
Reverted this back to internal